src/xtgeo/cube/_cube_import.py

```diff
@@ -314,24 +312,23 @@ def import_stormcube(self, sfile):  # pylint: disable=too-many-statements
             "Something when wrong in {}, code is {}".format(__name__, ier)
```

Comment:
Typo? Should perhaps be "Somehting went wrong....."  ?

Reply:
Fixed